versioned_docs/version-8.7/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -132,17 +132,92 @@ steps:
+| `IDENTITY_TENANTS_0_MEMBERS_0_USERNAME`      | If adding a **user** as a member of the tenant, the username. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_USERNAME`).                    | `user-name`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_GROUPNAME`     | If adding a **group** as a member of the tenant, the name of the group. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_GROUPNAME`).         | `group-name`  |
+| `IDENTITY_ENVIRONMENT_TENANTS_0_NAME`        | Needs description.                                                                                                                                                                                                                                 | Default       |
+| `IDENTITY_ENVIRONMENT_TENANTS_0_TENANTID`    | Needs description.                                                                                                                                                                                                                                 | `<default>`   |
+
+When converting the above variables for use in the Helm chart, ensure multiple tenants or members are added as a list. For example, a valid `application.yaml` configuration might look like the following:
+
+```
+identity:
+   tenants:
+    - name: Tenant Name
+      tenantId: tenant-name
+      members:
+        - type: GROUP
+          group-name: "Test Group"
+        - type: USER
+          username: test-user
+        - type: APPLICATION
+          application-id: test-application
+```
+
+## Configure clients
+
+The following variables can be used to configure [clients](/self-managed/identity/user-guide/additional-features/incorporate-applications.md). To add additional clients, increment the `0` value for each variable (for example, `KEYCLOAK_CLIENTS_1_NAME`).
+
+| Environment variable                                  | Description                                                           | Default value            |
+| ----------------------------------------------------- | --------------------------------------------------------------------- | ------------------------ |
+| `KEYCLOAK_CLIENTS_0_NAME`                             | A human-readable name for the client.                                 | zeebe                    |
+| `KEYCLOAK_CLIENTS_0_ID`                               | The client ID.                                                        | `${ZEEBE_CLIENT_ID}`     |
+| `KEYCLOAK_CLIENTS_0_SECRET`                           | The client secret.                                                    | `${ZEEBE_CLIENT_SECRET}` |
+| `KEYCLOAK_CLIENTS_0_TYPE`                             | The type of client, one of `CONFIDENTIAL`, `PUBLIC`, or `M2M`.        | `M2M`                    |
```

Comment:
The initial ticket only specified M2M, but it seems like all of these are available.

Reply:
[Correct](https://github.com/camunda-cloud/identity/blob/4b0b00105170daebdf65352d707b2b43995119d2/management-api/src/main/java/io/camunda/identity/record/ClientType.java#L11-L16)